### PR TITLE
chore(darkmode) - more colours

### PR DESCRIPTION
**Pull request by maltenuhn**
**Problem:**
We still had a lot of unthemed parts of the application that were amenable to the `useColorTheme` hook

**Fix:**
Adds the theme in more places

(NB this doesn't include icons, despite the branch name; I originally put the icon colours into the theme and then realised that complicated the issue and pulled that out into another branch, and then didn't change the branch name of this one.)

### CODE REVIEW

**Comment by maltenuhn** [comment]
This is ready for review, assuming the tests pass :)

File: editor/src/components/element-context-menu.tsx [hunk 1]
```diff
@@ -137,6 +137,7 @@ interface SelectableElementItemProps {
 }
 
 const SelectableElementItem = (props: SelectableElementItemProps) => {
+  const colorTheme = useColorTheme()
```
Comment (Rheeseyb) [inline editor/src/components/element-context-menu.tsx:140]: This appears to not actually be used?

File: editor/src/components/inspector/sections/component-section/component-section.tsx [hunk 2]
```diff
@@ -285,6 +284,7 @@ const RowForArrayControl = betterReactMemo(
       controlDescription,
     )
 
+    const colorTheme = useColorTheme()
```
Comment (Rheeseyb) [inline editor/src/components/inspector/sections/component-section/component-section.tsx:287]: This is also not used

File: editor/src/components/inspector/sections/component-section/component-section.tsx [hunk 3]
```diff
@@ -321,19 +321,15 @@ const RowForArrayControl = betterReactMemo(
             {propertyStatus.overwritable ? (
               <SquareButton highlight onMouseDown={toggleInsertRow}>
                 {insertingRow ? (
-                  <Icn
+                  <Icons.Minus
```
Comment (Rheeseyb) [inline editor/src/components/inspector/sections/component-section/component-section.tsx:324]: Was this change supposed to be in this PR?

File: editor/src/components/inspector/sections/component-section/component-section.tsx [hunk 4]
```diff
@@ -407,7 +403,7 @@ const RowForObjectControl = betterReactMemo(
         <InspectorSectionHeader>
           <SimpleFlexRow style={{ flexGrow: 1 }}>
             <PropertyLabel target={[propPath]} style={{ textTransform: 'capitalize' }}>
-              {warning}
+              {/* {warning} */}
```
Comment (Rheeseyb) [inline editor/src/components/inspector/sections/component-section/component-section.tsx:406]: Accidental commenting?

File: editor/src/components/inspector/sections/style-section/text-subsection/font-family-select-popup.tsx [hunk 5]
```diff
@@ -496,7 +504,7 @@ export const FontFamilySelectPopup = betterReactMemo(
           tabIndex={0}
           style={{
             ...style,
-            backgroundColor: 'white',
+            backgroundColor: colorTheme.inspectorBackground.value,
```
Comment (Rheeseyb) [inline editor/src/components/inspector/sections/style-section/text-subsection/font-family-select-popup.tsx:507]: This is inside a functional component so you could use the `useColorTheme` hook to get the color theme, which I believe is what you intended to do here but perhaps accidentally imported `colorTheme` (via autocomplete?) instead

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/element-context-menu.tsx (hunk 1 above)
Comment (maltenuhn) [reply editor/src/components/element-context-menu.tsx:140]: fixed

File: editor/src/components/inspector/sections/component-section/component-section.tsx (hunk 2 above)
Comment (maltenuhn) [reply editor/src/components/inspector/sections/component-section/component-section.tsx:287]: fixed

File: editor/src/components/inspector/sections/component-section/component-section.tsx (hunk 3 above)
Comment (maltenuhn) [reply editor/src/components/inspector/sections/component-section/component-section.tsx:324]: I initially thought we'd use the defaults on these, but in general I'm trying to use the `Icons` object, makes it easier to see where we're (not) being consistent.

File: editor/src/components/inspector/sections/component-section/component-section.tsx (hunk 4 above)
Comment (maltenuhn) [reply editor/src/components/inspector/sections/component-section/component-section.tsx:406]: huh, think so!

File: editor/src/components/inspector/sections/style-section/text-subsection/font-family-select-popup.tsx (hunk 5 above)
Comment (maltenuhn) [reply editor/src/components/inspector/sections/style-section/text-subsection/font-family-select-popup.tsx:507]: Yepp. Fixed

**Comment by maltenuhn** [comment]
Could I get another double eye on this?

**Review by enidemi — APPROVED** [review]

**Pull request merged**